scripts/firedrake-install

```diff
@@ -35,6 +35,9 @@ firedrake_apps = {
                 "git+ssh://github.com/icepack/icepack.git#egg=icepack"),
     "irksome": ("""Implicit Runge-Kutta methods. https://github.com/firedrakeproject/Irksome/""",
                 "git+ssh://github.com/firedrakeproject/Irksome.git#egg=Irksome"),
+    "femlium": ("""Interactive visualization of finite element simulations on geographic maps with folium.
+                   https://github.com/FEMlium/FEMlium/""",
```

Comment:
I think that this will result in a big space in the `--help` output, can you check? Also, since you have this lovely documentation, perhaps point at femlium.github.io rather than the source page?

Reply:
Thanks, I fixed that. I did that because I thought there were pep8 checks on the line length.

Reply:
Ah, we are a bit loose with line length restrictions. For future reference if you want one long string but to break it over multiple lines, then the python parse concatenates successive strings so you can write:

```python
string = ("abc "
          "def")
assert string == "abc def"
```